public/app/plugins/datasource/opentsdb/datasource.js

```diff
@@ -217,6 +218,13 @@ function (angular, _, dateMath) {
         dps.push([v, k * 1000]);
       });
 
+      tagKeys = Object.keys(md.tags);
+      _.each(md.aggregateTags, function(tag) {
+        tagKeys.push(tag);
+      });
+
+      target.updateTagKeys(tagKeys);
```

Comment:
Won't this cause exception if function is undefined  (edited has not been opened yet)